src/hooks/auth/useOauth2.ts

```diff
@@ -32,16 +32,13 @@ export const useOauth2 = ({
         OAuth2EnabledAppsInitialised
     );
 
-    const oAuthGrowthbookConfig = {
-        OAuth2EnabledApps: oAuth2EnabledApps as unknown as TOAuth2EnabledAppList,
-        OAuth2EnabledAppsInitialised,
-    };
-
-    const { OAuth2Logout: oAuthLogout } = useOAuth2(oAuthGrowthbookConfig, handleLogout ?? (() => Promise.resolve()));
-
     const logoutHandler = async () => {
         client?.setIsLoggingOut(true);
-        await oAuthLogout();
+        await OAuth2Logout({
+            redirectCallbackUri: `${window.location.origin}/callback`,
+            WSLogoutAndRedirect: handleLogout ?? (() => Promise.resolve()),
+            postLogoutRedirectUri: window.location.origin,
+        });
```

Comment:
`useOauth` from auth-client is deprecated as it does not include `id_token_hint` when logging out and does not clear stale states

Reply:
Just a question: Since handleLogout will likely clear the client tokens, after logout and get us to the origin url why do we need OIDC redirect us to the base route specified by postLogoutRedirectUri?